AE_PunktBeschriftung/include/io/InstanceReader.hpp

```diff
@@ -2,13 +2,13 @@
 
 #include <representations/Instance.hpp>
 #include <representations/Point.hpp>
-#include <representations/Solution.hpp>
+#include <representations/SolutionWithGraph.hpp>
 
 #include <string>
 
-void readInstance(const std::string& filename, Instance& instance, Solution& solution);
+void readInstance(const std::string& filename, Instance& instance, SolutionWithGraph& solution);
```

Comment:
Isn't the whole point of using an interface to use a Solution here, not the more specific one?

Reply:
Forgot to change that.